deployment/aws-terraform/README.md

```diff
@@ -0,0 +1,40 @@
+# Cloud One File Storage Security Terraform Deployment
+
+Cloud One File Storage Security stacks deployment by Terraform.
```

Comment:
What does Terraform do? What is its benefit? How does it improve life for the user?

Reply:
Terraform is an infrastructure as code tool for helping the user manage their cloud resources across different cloud providers. The user doesn't have to learn a specific cloud provider's tool to deploy and manage their cloud resources. The initiative of the plugin is from the users, who want to deploy FSS stacks by Terraform instead of cloud formation template.

Reply:
Deploy File Storage Security stacks across different cloud providers with Terraform.